chat-plugins/misc.js

```diff
@@ -176,14 +176,22 @@ exports.commands = {
 	},
 
 	roomlist: function (target, room, user) {
-		if (!this.can('roomowner')) return;
+		//if (!this.can('roomowner')) return;
 
+		if (!this.can('roomowner')) {
+		
+		let header = ['<b><font color="#1aff1a" size="2">Total users connected: ' + Rooms.global.userCount + '</font></b><br />'],
```

Comment:
This is a bad idea, you should let all the variables be declared and check permissions later (imo).